test_themes/tests/test_new_page_templates.py

```diff
@@ -190,6 +190,13 @@ def check(theme_name, website):
                                 conflict.difference_update(white_list)
                                 if len(conflict) > 1:
                                     errors.append("Using %r, view %r contains conflicting classes: %r in %r (according to pattern %r)" % (theme_name, view.key, conflict, classes, conflicting_classes_re.pattern))
+                        for el in html_tree.xpath('//*[@style]'):
+                            styles = el.attrib['style'].split(';')
+                            properties = list(map(lambda x: x.split(':')[0].strip(),
+                                filter(lambda x: x, styles)
+                            ))
+                            if len(properties) != len(set(properties)):
+                                errors.append("Using %r, view %r contains duplicate style properties: %r" % (theme_name, view.key, el.attrib['style']))
```

Comment:
It could be useful to hint at which property is duplicated, would that be easy to do?

Reply:
I did that initially, but it was actually more helpful when fixing issues to see the actual values, to know what was supposed to have been removed. And since it is not supposed to be drawn, I chose to keep it simple.